pkg/option/config.go

```diff
@@ -434,6 +434,9 @@ const (
 	// ToFQDNsMinTTL is the minimum time, in seconds, to use DNS data for toFQDNs policies.
 	ToFQDNsMinTTL = "tofqdns-min-ttl"
 
+	// ToFQDNsActiveConnectionsTTL is the time for which connections should be idle for after the DNS TTL expires
```

Comment:
For the user-facing flag, I think it would help to define not just that the connection "should be idle for X long", but what the agent will do with entries based on this timer.

For instance:
```suggestion
	// ToFQDNsActiveConnectionsTTL is the time after which idle DNS entries will be garbage collected, following TTL expiry
```

(Though strictly speaking I don't think there's any bounding to say how quickly those entries _will_ be expired after this timeout, only that this sets a minimum period after which the timeout will occur).